qa/tasks/cephfs/test_volume_client.py

```diff
@@ -703,12 +696,7 @@ def test_get_authorized_ids(self):
             guest_entity_2=guest_entity_2,
         )))
         # Check the list of authorized IDs and their access levels.
-        if self.py_version == 'python3':
-            expected_result = [('guest1', 'rw'), ('guest2', 'r')]
-        else:
-            expected_result = [(u'guest1', u'rw'), (u'guest2', u'r')]
-
-        self.assertItemsEqual(str(expected_result), auths)
+        self.assertItemsEqual("[('guest1', 'rw'), ('guest2', 'r')]", auths)
```

Comment:
We get this error here:

````
2019-05-08T05:33:26.981 INFO:tasks.cephfs_test_runner:======================================================================
2019-05-08T05:33:26.981 INFO:tasks.cephfs_test_runner:FAIL: test_get_authorized_ids (tasks.cephfs.test_volume_client.TestVolumeClient)
2019-05-08T05:33:26.981 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2019-05-08T05:33:26.982 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2019-05-08T05:33:26.982 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-pdonnell-testing-20190501.220632/qa/tasks/cephfs/test_volume_client.py", line 699, in test_get_authorized_ids
2019-05-08T05:33:26.982 INFO:tasks.cephfs_test_runner:    self.assertItemsEqual("[('guest1', 'rw'), ('guest2', 'r')]", auths)
2019-05-08T05:33:26.982 INFO:tasks.cephfs_test_runner:AssertionError: Element counts were not equal:
2019-05-08T05:33:26.982 INFO:tasks.cephfs_test_runner:First has 2, Second has 6:  'u'
````

From: /ceph/teuthology-archive/pdonnell-2019-05-08_04:00:30-fs-wip-pdonnell-testing-20190501.220632-distro-basic-smithi/3939463/teuthology.log

Keep in mind that the machine running the test is always python 2.7 (until we eventually switch teuthology to py3). So, I think the result should always use the unicode strings?